appveyor.yml

```diff
@@ -80,15 +80,15 @@ test_script:
             | Select-String -Pattern '<test-run id="[^"]*"\s+testcasecount="(\d+)"\s+result="([^"]*)"\s+total="(\d+)"\s+passed="(\d+)"\s+failed="(\d+)".*' `
             | foreach {
                 $result = $_.Matches.Groups
-                #$result[0].Value
+
                 $total  = $result[1].Value
                 $passed = $result[4].Value
                 $failed = $result[5].Value
                 $notPassed = $total - $passed
                 $testErrors = $failed
                 $testName = $resultFilePath.Split('\\')[-3]
                 $artifactName = "TestResult#${testRun}.${testErrors}err.${testName}"
-                Push-AppveyorArtifact $resultFilePath -FileName "${artifactName}.xml"
```

Comment:
I agree, the artifacts have become quite noisy.
At least on error, this file should be kept because it contains the most complete information.
(The best solution IMO would be one `.xml` with all the test results.)
An alternative could be to zip them into one archive -- with the drawback that it cannot be opened directly.

Reply:
I think I may have asked before, but what's the benefit of these xml for user? 
The main reason we have them for opencover/appveyor to report the stats. As a user a I look at the *.log files.
Merging xmls together is a time consuming operation which will result in a tens of thousands of lines of xml....